include/libopencm3/stm32/adc.h

```diff
@@ -36,6 +36,8 @@
 #       include <libopencm3/stm32/l1/adc.h>
 #elif defined(STM32L4)
 #       include <libopencm3/stm32/l4/adc.h>
+#elif defined(STM32L4PLUS)
+#       include <libopencm3/stm32/l4/adc.h>
```

Comment:
I don't like this approach. I think it's why you landed in problems with the VPATH.  I don't like the idea of l4 plus doing it's own thing for shared code vs all the other stm32's shared code.